en/pocket_privacy_policy_eu.md

```diff
@@ -0,0 +1,124 @@
+
+## Phishing {: #phishing }
+
+Identity theft and the practice currently known as "phishing" are of great concern to Pocket. Safeguarding information to help protect you from identity theft is a top priority. We do not and will not, at any time, request your username, password, credit card information or national identification number in an unsolicited e-mail or telephone communication. For more information about phishing, visit the Federal Trade Commission’s website.
+
+## Managing My Privacy and Email Settings {: #opt-out }
+
+You can access, correct, or delete your personal information and export your data from your Pocket Profile. You can also view and control which third party services, if any, you want to connect to your Pocket account. These settings are at: <https://getpocket.com/options>
+
+As part of our default settings to share saved and recommended content, we send emails to our users. You may opt-out of receiving these, and updates on our service and your use, by using the unsubscribe feature on any of our email correspondence or from your account settings. We will try to communicate with you only according to your instructions, although it may take some time to process your request.
+
+## Amendments to Policy {: #amendments }
+
+Please note this Privacy Policy will change from time to time. We may amend this Privacy Policy at any time by posting the amended terms on our Website that you can access at [getpocket.com](https://getpocket.com/) or through the options or help menu of the Pocket application. All amended terms shall automatically be effective thirty (30) days after they are initially posted on the Website. If we make any material changes to our Privacy Policy that affect user information already stored in our database, we will post a prominent notice on our Website notifying users that our Privacy Policy has changed. Our use of your information shall be governed by the terms and conditions of our Privacy Policy in effect when you first use our application or thirty (30) days after the terms are changed, as explained above.
+
+## Contact {: #contact }
+
+If you have questions or concerns regarding this Privacy Policy, you should contact us at [privacy@getpocket.com](mailto:privacy@getpocket.com) or [https://help.getpocket.com/contact](https://help.getpocket.com/contact).
+
+If you have any other questions regarding personal data or our privacy practices, please contact us at [dpo@mozilla.com](mailto:dpo@mozilla.com), or you can contact our DPO by mail at:
+
+Bird & Bird DPO Services SRL\
```

Comment:
While this works in GitHub, it's not how line breaks are created in other documents. Will switch to 2 spaces like in other documents.